README.md

```diff
@@ -164,14 +164,26 @@ Indy Catalyst components:
 
 ## Key Technical Elements
 
+For a quickstart guide on running the indy-catalyst project in development on your local machine follow the guide located [here](docs/README.md)
+
 ### Credential Registry
 
 TODO
 
 Credential Registry provides a set of RESTful web services you can use to query data from your third-party application, an introduction to use of these API's is available [here](docs/IndyCat-REST-Services.md).
 
+The credential registry is a controller for the issuer-verifier agent in context of Indy Catalyst.
```

Comment:
This line is not correct. Please replace "the issuer-verifier" with "the holder-prover" it will be a sufficient correction.

Reply:
@swcurran  Good catch, I got it right in the description of the actually issuer-verifier.. must have had a brain cramp.
https://github.com/bcgov/indy-catalyst/pull/401/commits/59f95ab573037e76e80a3e8085f0ba5635473ff4